docs/content/concepts/ops-jobs-graphs/metadata-tags.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Metadata & Tags | Dagster
+description: Metadata and tags provide two ways to attach information in Dagster
+---
+
+# Metadata & Tags
+
+Metadata and tags provide two different ways for you to attach information to your jobs and the runs launched from those jobs. The main difference between metadata and tags is what "object" the information is attached to. Metadata is attached to JobDefinitions (specified using the @job decorator) and tags are attached to runs that are created by executing a job.
+
+## Metadata
+
+Metadata allows you to attach information to a job. This information can be whatever you want, but possible cases include:
+
+- keeping track of the team responsible for maintaining a job
+- linking to documentation or other resources
+- displaying the git hash corresponding to the current job definition
+
+Note: If you specify metadata on your jobs, you will need to ensure that you Dagit version is >=0.14.18
```

Comment:
@chris does this note sound ok?

Reply:
I would link to the deployment docs here so that people don't get confused what dagit version means. So maybe say something like, if running dagster using separate dagit and user code installations (more info here[link]), then ensure dagit installation is >=...